client/lib/signup/step-actions/index.js

```diff
@@ -437,28 +439,110 @@ export function setOptionsOnSite( callback, { siteSlug, siteTitle, tagline } ) {
 	);
 }
 
-export function setStoreFeatures( callback, { siteSlug } ) {
+export async function setStoreFeatures( callback, { siteSlug }, stepProvidedItems, reduxStore ) {
 	if ( ! siteSlug ) {
 		defer( callback );
 		return;
 	}
 
-	//Hard-code the theme slug for now, we'll use design selection step in a future iteration.
+	try {
+		/*
+		 * Get the block pattern source for use in our new home page.
+		 * Ideally we'd pull this content from the patterns repo dynamically, but we don't
+		 * have a way to get raw block source.
+		 * Original pattern: https://dotcompatterns.wordpress.com/wp-admin/post.php?post=4348&action=edit
+		 */
+		const patternPost = `
+			<!-- wp:columns {"align":"wide"} -->
+			<div class="wp-block-columns alignwide">
+				<!-- wp:column -->
+				<div class="wp-block-column">
+					<!-- wp:image -->
+					<figure class="wp-block-image"><img alt="" /></figure>
+					<!-- /wp:image -->
+				</div>
+				<!-- /wp:column -->
+
+				<!-- wp:column {"style":{"spacing":{"padding":{"right":"10%","left":"5%"}}}} -->
+				<div class="wp-block-column" style="padding-right:10%;padding-left:5%">
+					<!-- wp:heading {"textAlign":"left","fontSize":"x-large"} -->
+					<h2 class="has-text-align-left has-x-large-font-size" id="item-name">${ translate(
+						'Item Name'
+					) }</h2>
+					<!-- /wp:heading -->
+
+					<!-- wp:paragraph {"align":"left","style":{"color":{"text":"#808080"}},"fontSize":"small"} -->
+					<p class="has-text-align-left has-text-color has-small-font-size" style="color:#808080">${ translate(
+						'Quick details'
+					) }</p>
+					<!-- /wp:paragraph -->
+
+					<!-- wp:heading {"level":3,"fontSize":"medium"} -->
+					<h3 class="has-medium-font-size" id="0-00">$0.00</h3>
+					<!-- /wp:heading -->
+
+					<!-- wp:jetpack/recurring-payments -->
+					<div class="wp-block-jetpack-recurring-payments">
+						<!-- wp:jetpack/button {"element":"a","uniqueId":"recurring-payments-id","text":"${ translate(
+							'Buy Now'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **76** times:
`translate( 'Buy now' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/seller-experience-add-blocks-to-frontpage)